tests/infra/runner.py

```diff
@@ -93,20 +95,27 @@ def run(get_command, args):
 
         command_args = get_command_args(args, get_command)
 
-        nodes = filter_nodes(primary, backups, args.send_tx_to)
+        nodes_to_send_to = filter_nodes(primary, backups, args.send_tx_to)
         clients = []
         client_hosts = []
-        if args.num_localhost_clients:
-            client_hosts = ["localhost"] * int(args.num_localhost_clients)
+        if args.one_client_per_backup:
```

Comment:
I think these options should all be additive, since we don't document their dependencies. So `--one-client-per-backup --num-localhost-clients 3 --client-nodes 1.2.3.4` should spawn 3 localhost + 1 on `1.2.3.4` + 1-per-backup.

Reply:
Ah I've just seen that some of these are currently mutually exclusive, so the code almost matches the arg spec, but we `--num-localhost-clients` is also exclusive with `--one-client-per-backup` in this code. Don't think they need to be or should be exclusive.

Reply:
The two options (`--one-client-per-backup` and `--num-localhost-clients`) are now non mutually exclusive.